conf/test_full.config

```diff
@@ -35,10 +35,10 @@ params {
 
     // Panelprep optional args
     remove_samples = "NA12878,NA12891,NA12892"
-    normalize      = true
-    compute_freq   = true
-    phase          = true
+    normalize      = false
+    compute_freq   = false
+    phase          = false
 
     // Impute tools
-    tools = "glimpse1"
+    tools = "glimpse2"
```

Comment:
Why this change?

Reply:
It is to reduce the computational time and cost, as glimpse1 has more step and takes more time to compute.